test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -977,6 +992,40 @@ describe('33acrossBidAdapter:', function () {
       });
     });
 
+    context('when Global Placement ID (gpid) is defined', function() {
+      let bidderRequest;
+
+      beforeEach(function() {
+        bidderRequest = {};
+      });
+
+      it('passes the Global Placement ID (gpid) in the request', function() {
+        const ttxRequest = new TtxRequestBuilder()
+          .withBanner()
+          .withProductAndGPID('siab', 'fakeGPID0')
+          .build();
+        const serverRequest = new ServerRequestBuilder()
+          .withData(ttxRequest)
+          .build();
+
+        let copyBidRequest = utils.deepClone(bidRequests);
+        const addGPIDBidRequests = copyBidRequest.map(function(bidRequest, index) {
```

Comment:
just the naming is a bit confusing, usually functions start with a verb.. in this case `addGPIDBidRequests` is not a function, so my suggestion is to use a name like `bidRequestsWithGpid`